modin/pandas/series.py

```diff
@@ -1068,6 +1068,27 @@ def slice_shift(self, periods=1, axis=0):
                 )
             )
 
+    def unstack(self, level=-1, fill_value=None):
+        """
+        Unstack, a.k.a. pivot, Series with MultiIndex to produce DataFrame.
+        The level involved will automatically get sorted.
+        Parameters
+        ----------
+        level : int, str, or list of these, default last level
+            Level(s) to unstack, can pass level name.
+        fill_value : scalar value, default None
+            Value to use when replacing NaN values.
+        Returns
+        -------
+        DataFrame
+            Unstacked Series.
+        """
+        from .dataframe import DataFrame
+
+        return DataFrame(
+            query_compiler=self._query_compiler.unstack(level, fill_value)
+        ).droplevel(0, axis=1)
+
```

Comment:
can we add a check here, that number of levels at the column axis is more than 1?

Reply:
@dchigarev Index of this `DataFrame` is `MultiIndex` with minimum 2 levels (the upper level is ['__reduced__']). I think this check will be excess here.

Reply:
isn't this backend-specific?

Reply:
Yes, this is backend-specific. Check was added.